cmd/jaeger/config-badger.yaml

```diff
@@ -1,12 +1,16 @@
 service:
-  extensions: [jaeger_storage, jaeger_query]
+  extensions: [jaeger_storage, jaeger_query, jaeger_static_sampling]
   pipelines:
     traces:
       receivers: [otlp]
       processors: [batch]
       exporters: [jaeger_storage_exporter]
 
 extensions:
+  jaeger_static_sampling:
```

Comment:
There should be no name like static sampling. The extension name is remotesampling